msgraph-connector-test/src/com/microsoft/graph/GraphTestClient.java

```diff
@@ -0,0 +1,36 @@
+package com.microsoft.graph;
+
+import java.util.ArrayList;
+import java.util.UUID;
+
+import com.microsoft.auth.OAuth2Feature;
+
+import ch.ivyteam.ivy.application.IApplication;
+import ch.ivyteam.ivy.rest.client.RestClient;
+import ch.ivyteam.ivy.rest.client.RestClients;
+import ch.ivyteam.ivy.rest.client.mapper.JsonFeature;
+import ch.ivyteam.ivy.rest.client.security.CsrfHeaderFeature;
+
+public class GraphTestClient {
+
+  public static void mockForApp(IApplication app) {
+    RestClients clients = RestClients.of(app);
+    var graphMock = clients.find(UUID.fromString("007036dc-72d1-429f-88a7-ba5d5cf5ae58"))
+      .toBuilder()
+      .uri(GraphServiceMock.URI)
+      .feature(JsonFeature.class.getName())
+      .feature(CsrfHeaderFeature.class.getName())
+      .property("AUTH.baseUri", GraphAuthMock.URI)
+      .property("AUTH.secretKey", "1")
+      .property("scope", "user.read calendars.read")
+      .toRestClient();
+
+    var features = new ArrayList<>(graphMock.features());
+    features.remove(OAuth2Feature.class.getName());
+    graphMock = new RestClient(graphMock.uri(), graphMock.name(), graphMock.uniqueId(), graphMock.description(),
+      features, graphMock.properties(), graphMock.metas());
+
+    clients.set(graphMock);
+  }
```

Comment:
would be nice to move it here @alexsuter 
... but now we have to re-define everything, just because we can't remove a feature.